ov/server_hardware.go

```diff
@@ -689,3 +704,35 @@ func (c *OVClient) SetPowerState(serverHardwareId string, powerState map[string]
 
 	return nil
 }
+
+// Delete the rack server using URI
+func (c *OVClient) DeleteServerHardware(uri utils.Nstring) error {
+	var (
+		hardware ServerHardware
+		err      error
+	)
+
+	hardware, err = c.GetServerHardwareByUri(uri)
+	if err != nil {
+		return err
+	}
+	if hardware.Name != "" {
+		log.Infof("URI:%s", hardware.URI)
+		log.Debugf("REST : %s \n %+v\n", hardware.URI, hardware)
+
+		if hardware.URI == "" {
+			log.Warn("Unable to post delete, no uri found.")
+			return err
+		}
+		_, err := c.RestAPICall(rest.DELETE, hardware.URI.String(), nil)
+		if err != nil {
+			log.Errorf("Error submitting server hardware delete request: %s", err)
+			return err
+		}
+
+		return nil
```

Comment:
can you check if deletion returns any task uri...if not then the current implementation is fine

Reply:
Done